src/pb-app/src/checkout-form/index.jsx

```diff
@@ -1,41 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {ElementsConsumer, CardElement} from '@stripe/react-stripe-js';
 
 import CardSection from '../card-section/index.jsx';
 import ButtonBase from '../button-base/index.jsx';
 import './checkout-form.css';
 
-class CheckoutForm extends React.Component {
+async function paymentSetup(amount) {
+  const response = await fetch('https://api.piggybank.lukejoshuapark.io/payment/setup', {
+    method: 'POST',
+    mode: 'cors', 
+    headers: {
+      'content-type': 'application/json'
+    },
+    redirect: 'follow',
+    referrerPolicy: 'no-referrer',
+    body: JSON.stringify({amount: parseFloat(amount)})
+  });
+  const responseModel =  await response.json();
+  if (response.status !== 200 && !responseModel.error) {
+    responseModel.error = "Internal Server Error :(";
```

Comment:
Minor - but messaging here might be better as "Something went wrong" etc. - it may not be an internal server error e.g. we'll end up here if the client loses network connectivity, for example.